errbot/plugin_manager.py

```diff
@@ -63,6 +63,10 @@ def install_packages(req_path: Path):
 
         Return an exc_info if it fails otherwise None.
     """
+    def is_docker():
+        with open('/proc/1/cgroup') as d:
+            return 'docker' in d.read()
```

Comment:
In poking at this `/proc/1/cgroup` file, it seems (to me) probably safer to check for `/docker/` vs just `docker`.
I don't know the inner-workings of a container or docker but I just wondering if there is a scenario, like docker in docker or mounting a disk/cgroup where this could be a false positive.

Reply:
@sijis Solid concern. Let me do some testing. The /proc/1 pid is always the `init` process so it should always be the base filesystem. In this case, a non-docker container would be `/` so I'm not too worried about mounting a disk/cgroup but will double check docker in docker.